external/elasticsearch/src/main/java/com/digitalpebble/stormcrawler/elasticsearch/persistence/AggregationSpout.java

```diff
@@ -255,8 +250,8 @@ public void onResponse(SearchResponse response) {
                 if (hitsForThisBucket == 1 && !iterator.hasNext()) {
                     String strDate = (String) keyValues.get("nextFetchDate");
                     try {
-                        mostRecentDateFound = formatter.parse(strDate);
-                    } catch (ParseException e) {
+                        mostRecentDateFound = Instant.parse(strDate);
```

Comment:
@jnioche it seems working with your proposal... I will be running some test in my cluster to guarantee that everything is working as expected... will add the updates here

Reply:
@jcruzmartini how is it going?

Reply:
@jnioche I run an end to end test this morning... everything is looking great. this PR is ready to be merged. thanks